domain/src/main/java/org/oppia/domain/question/QuestionConstantsProvider.kt

```diff
@@ -0,0 +1,23 @@
+package org.oppia.domain.question
+
+import dagger.Module
+import dagger.Provides
+import javax.inject.Qualifier
+
+/** Provider to return any constants required during the training session. */
```

Comment:
Perhaps this should be on the Module instead of this annotation?

Reply:
Done